lars-reimann/api-editor
# feat: Added undo and redo buttons for annotation editing

Closes #535.

### Summary of Changes

Added undo and redo buttons which can revert or redo changes related to annotations.
The "editing history" is up to 10 changes.

### Screenshots (if necessary)
![image](https://user-images.githubusercontent.com/2046129/174303423-6a3e728d-fb83-414f-a6d3-a0912f7e0dcf.png)

### Testing Instructions
Add or change annotations. The changes should be reverted by clicking the "Undo" button. The reverted changes can than be restored by clicking the "Redo" button.
